### DISCUSSION TITLE
Use `throttle_classes` for spyne application for set rate limit

### DISCUSSION THREAD

**Discussion by chiragkanhasoft** [arskom/spyne#653] [Q&A]
I want to use throttle_classes for my API which has spyne application, i found below link to implement it in url.py file, but it is not working for spyne DjangoView.

Link for use classes in URL

[https://stackoverflow.com/questions/60666418/remove-authentication-and-permission-for-specific-url-path](https://stackoverflow.com/questions/60666418/remove-authentication-and-permission-for-specific-url-path)

url(
r"^vapi/",
(permission_classes([IsAuthenticated])(vapi_view.vapp)).as_view(application=vapi_view.vapp),
),

**Comment by plq** [comment]
I never used django so I don't know, really.

Does the django example in the repository help? 

https://github.com/arskom/spyne/tree/master/examples/django

**Reply by chiragkanhasoft** [reply]
@plq , this is not helpful to me

**Comment by plq** [comment]
If this is a Django issue, please seek help in Django's support channels. If this is a Spyne issue, please reply with a working code sample that reproduces the issue, its output and what makes you think that the code "doesn't work" in that output.

**Comment by chiragkanhasoft** [comment]
@plq , this is not a django issue, how to use decorator with  spyne Service.

**Comment by chiragkanhasoft** [comment]
@plq ,
This is spyne Django View, which is not working.
```
url(
r"^vapi/",
(permission_classes([IsAuthenticated])(vapi_view.vapp)).as_view(application=vapi_view.vapp),
),

```
This is django's django view which is working
`url(r'^test/$', (permission_classes([IsAuthenticated])(algo_program_views.TestView)).as_view()),`
